.github/workflows/black.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: black
```

Comment:
The linters workflow will take care of the 'black' check. I do not think we need a separate workflow for black.